### PR TITLE
feat/AB#82794_Change-delete-icon-to-be-red-and-drag-drop-to-be-grey

### PR DESCRIPTION
**Pull request by TaiKamilla**
# Description

Fix button variant logic in navbar component The commit fixes a logic issue in the navbar component's HTML template where the button variant was incorrectly set based on the vertical flag. This has been corrected to ensure that the correct button variant is applied.

db2d43

## Useful links

- [Link to ticket](https://dev.azure.com/WHOHQ/EMSSAFE/_workitems/edit/82794)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Improvement (refactor or addition to existing functionality)

## Screenshots

Screenshot 2024-01-03 at 13 09 27

# Checklist:

( * == Mandatory ) 

- [x] * I have set myself as assignee of the pull request
- [x] * My code follows the style guidelines of this project
- [x] * Linting does not generate new warnings
- [x] * I have performed a self-review of my own code
- [x] * I have put the ticket for review, adding the oort-frontend team to the list of reviewers
- [ ] * I have commented my code, particularly in hard-to-understand areas
- [ ] * I have put JSDoc comment in all required places
- [x] * My changes generate no new warnings
- [x] * I have included screenshots describing my changes if relevant
- [x] * I have selected labels in the Pull Request, according to the changes with code brings
- [ ] I have made corresponding changes to the documentation ( if required )
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/shared/src/lib/components/navbar/navbar.component.html [hunk 1]
```diff
@@ -103,7 +103,7 @@
                 >S
               </ui-icon>
               <ui-button
-                [variant]="!vertical ? 'danger' : 'primary'"
+                [variant]="vertical ? 'danger' : 'primary'"
```
Comment (AntoineRelief) [inline libs/shared/src/lib/components/navbar/navbar.component.html]: @TaiKamilla 
how does it look like when it's horizontal with this change?

Comment (AntoineRelief) [reply libs/shared/src/lib/components/navbar/navbar.component.html]: I think you can just use the same value for both

Comment (TaiKamilla) [reply libs/shared/src/lib/components/navbar/navbar.component.html]: done!

**Comment by unai-reliefapp** [comment]
@AntoineRelief  Do we have to keep same styling for drag handlers and remove icons in the whole app for consistency?
In that case this one is missing the grey variant: https://github.com/ReliefApplications/ems-frontend/blob/2.x.x/libs/shared/src/lib/components/widgets/common/sorting-settings/sorting-settings.component.html#L100-L106
And these deletes are missing the danger variant:
https://github.com/ReliefApplications/ems-frontend/blob/2.x.x/libs/shared/src/lib/components/role-summary/role-auto-assignment/role-auto-assignment.component.html#L46-L56
https://github.com/ReliefApplications/ems-frontend/blob/2.x.x/libs/shared/src/lib/components/templates/templates.component.html#L68

**Comment by AntoineRelief** [comment]
@unai-reliefapp 
All agreed!

**Review by unai-reliefapp — CHANGES_REQUESTED** [review]
https://github.com/ReliefApplications/ems-frontend/pull/2236#issuecomment-1875503769

**Review by unai-reliefapp — APPROVED** [review]

**Pull request merged**